### DISCUSSION TITLE
Width of new settings window too wide

### DISCUSSION THREAD

**Discussion by Hk1020** [jamulussoftware/jamulus#1659] [Technical Support & Help]
**Describe the bug**
The new tabbed settings window could open much narrower. It doesn’t need the space anymore of the old version (which I still prefer). Does it maybe reuse an old saved setting from an older client?

**To Reproduce**
Install the 3.8.0.beta and click settings. 

**Expected behavior**
On first use the window should open as small as possible.

**Comment by softins** [comment]
The size is saved in the ini file. It is very easy, on first opening after updating, to resize the settings window to the desired size, and this size will be saved and used for all future occasions.

It would be much more complicated to expect the program to notice that the stored value was from an earlier version of the program and to ignore it on just that one occasion.

The default size for the window appears to be correct when no stored size is available, e.g. when the ini file does not exist:
```
Jamulus -i temp.ini
```
